contracts/core/RevenuePath.sol

```diff
@@ -709,13 +700,13 @@ contract RevenuePath is Ownable, Initializable, ReentrancyGuard {
     /** @notice Get the platform wallet address
      */
     function getPlatformWallet() external view returns (address) {
-        return platformFeeWallet;
+        return IReveelMain(mainFactory).getPlatformWallet();
     }
 
     /** @notice Get the platform fee percentage
      */
-    function getPlatformFee() external view returns (uint256) {
-        return platformFee;
+    function getPlatformFee() external view returns (uint88) {
+        return IReveelMain(mainFactory).getPlatformFee();
```

Comment:
We can not do this. :negative_squared_cross_mark: 

The fee should always remain constant when a revenue path is launched. This will enable us to change fees at any point.

Reply:
yea that's fair enough. I will change this one back & pack it in with the other vars.